pkg/cpi/commonUtils.go

```diff
@@ -190,3 +191,32 @@ func (httpFileUploadRequestParameters HttpFileUploadRequestParameters) HandleHTT
 	}
 	return errors.Errorf("%s, Response Status code: %v", httpFileUploadRequestParameters.ErrMessage, response.StatusCode)
 }
+
+// HandleHTTPGetRequestResponse - Handle the GET Request response data
+func (httpGetRequestParameters HttpFileUploadRequestParameters) HandleHTTPGetRequestResponse() (string, error) {
```

Comment:
Is there not a high potential of reducing code repetition here?

Reply:
this function has more or than one return values, being returned many times.  so code duplication not applicable.

Reply:
in the other cases the return value could be an empty string and then ignored by the caller:
`_, err = HandleHTTPGetRequestResponse()`

Reply:
empty response could also means no api proxies exist in the API portal i feel

Reply:
And that is not an error case?

We decided to approach this with a separate PR. So let's discuss it when we are there.